dev/deps/spark-deps-hadoop-2.7

```diff
@@ -152,6 +152,7 @@ minlog-1.3.0.jar
 netty-3.9.9.Final.jar
 netty-all-4.1.30.Final.jar
 objenesis-2.5.1.jar
+okapi-shade-0.4.1.jar
```

Comment:
Finally, it's good to have a single file although this is huge (17MB, top 1 in terms of the size).
```
$ ls -alhS assembly/target/scala-2.12/jars/* | head
-rw-r--r--  1 dhyun  staff    17M Jun  2 12:19 assembly/target/scala-2.12/jars/okapi-shade-0.4.1.jar
-rw-r--r--  1 dhyun  staff    13M Jun  2 12:19 assembly/target/scala-2.12/jars/breeze_2.12-0.13.2.jar
-rw-r--r--  1 dhyun  staff    11M Jun  2 12:19 assembly/target/scala-2.12/jars/aws-java-sdk-1.7.4.jar
-rw-r--r--  1 dhyun  staff    11M Jun  2 12:19 assembly/target/scala-2.12/jars/hive-exec-1.2.1.spark2.jar
-rw-r--r--  1 dhyun  staff    10M Jun  2 12:19 assembly/target/scala-2.12/jars/scala-compiler-2.12.8.jar
-rw-r--r--  1 dhyun  staff   9.2M Jun  2 12:19 assembly/target/scala-2.12/jars/spire_2.12-0.13.0.jar
-rw-r--r--  1 dhyun  staff   8.0M Jun  2 12:19 assembly/target/scala-2.12/jars/hadoop-hdfs-2.7.4.jar
-rw-r--r--  1 dhyun  staff   7.2M Jun  2 12:19 assembly/target/scala-2.12/jars/spark-catalyst_2.12-3.0.0-SNAPSHOT.jar
-rw-r--r--  1 dhyun  staff   6.5M Jun  2 12:19 assembly/target/scala-2.12/jars/spark-core_2.12-3.0.0-SNAPSHOT.jar
-rw-r--r--  1 dhyun  staff   5.8M Jun  2 12:19 assembly/target/scala-2.12/jars/spark-sql_2.12-3.0.0-SNAPSHOT.jar
```

I'm wondering if we need the following meta files in `okapi-shade-0.4.1.jar`. Can we remove them from `okapi-shade-0.4.1`.jar?
```
$ jar tvf assembly/target/scala-2.12/jars/okapi-shade-0.4.1.jar | awk '{print $8}' | sort | more
...
META-INF/maven/org.apache.commons/
META-INF/maven/org.apache.commons/commons-lang3/
META-INF/maven/org.apache.commons/commons-lang3/pom.properties
META-INF/maven/org.apache.commons/commons-lang3/pom.xml
META-INF/maven/org.apache.logging.log4j/
META-INF/maven/org.apache.logging.log4j/log4j-api-scala_2.12/
META-INF/maven/org.apache.logging.log4j/log4j-api-scala_2.12/pom.properties
META-INF/maven/org.apache.logging.log4j/log4j-api-scala_2.12/pom.xml
META-INF/maven/org.apache.logging.log4j/log4j-api/
META-INF/maven/org.apache.logging.log4j/log4j-api/pom.properties
META-INF/maven/org.apache.logging.log4j/log4j-api/pom.xml
...
```

Reply:
We'll remove the unnecessary meta files.